src/pkg/cli/deploymentsList.go

```diff
@@ -37,20 +35,19 @@ func DeploymentsList(ctx context.Context, loader client.Loader, client client.Gr
 	}
 
 	// map to Deployment struct
-	deployments := make([]PrintDeployment, numDeployments)
-	for i, d := range response.Deployments {
-		actionText := strings.TrimPrefix(d.Action.String(), "DEPLOYMENT_ACTION_")
-		if d.Action == defangv1.DeploymentAction_DEPLOYMENT_ACTION_UNSPECIFIED {
-			actionText = "UP"
+	deployments := make([]PrintDeployment, 0, numDeployments)
+	for _, d := range response.Deployments {
+
+		if d.GetAction() == defangv1.DeploymentAction_DEPLOYMENT_ACTION_DOWN {
```

Comment:
We can't write the query to exclude these before we get here?

Reply:
Probably could but (to me) it makes sense to have the CLI filter as it's the "presentation" layer.

Reply:
I disagree. Client side filtering results in non-deterministic result count. We ask for 10, but you may filter out half of them (or more!). There's no reason to return results from dynamo to fabric, and from fabric to the cli that will never be presented.